src/components/review/components/reviews-list.jsx

```diff
@@ -22,11 +26,14 @@ class ReviewsList extends PureComponent {
           </span>
         </h2>
         <ul className="reviews__list">
-          {reviews.map((review) => {
-            return (<ReviewsItem key={review.id} review={review} />);
+          {reviewsRevert.map((review, index) => {
+            if (index < maxReviews) {
+              return (<ReviewsItem key={review.id} review={review} />);
+            }
+            return null;
```

Comment:
Можно записать короче (но необязательно, если вызывает трудности в чтении кода):

```suggestion
              return index < maxReviews && (<ReviewsItem key={review.id} review={review} />);
            }
```